src/config/build_config.py

```diff
@@ -1,7 +1,7 @@
 # Frameworks for which you want to disable both builds and tests
-DISABLE_FRAMEWORK_TESTS = []
+DISABLE_FRAMEWORK_TESTS = ["tensorflow", "pytorch"]
 # Disable new builds or build without datetime tag
 DISABLE_DATETIME_TAG = False
 # Note: Need to build the images at least once with DISABLE_DATETIME_TAG = True
 # before disabling new builds or tests will fail
-DISABLE_NEW_BUILDS = False
+DISABLE_NEW_BUILDS = True
```

Comment:
Sequence of events to use DISABLE_DATETIME_TAG and DISABLE_NEW_BUILDS:

1. Run a build with `DISABLE_DATETIME_TAG = True`, and `DISABLE_NEW_BUILDS = False`
2. Push all subsequent commits (while running PR tests) with `DISABLE_DATETIME_TAG = True` and `DISABLE_NEW_BUILDS = True`.
3. When tests are confirmed to be working, set both to False, and revert build_config and test_config to default state.